pkg/k8shandler/status.go

```diff
@@ -166,6 +167,7 @@ func translatePodMap(podStateMap elasticsearch.PodStateMap) logging.PodStateMap
 
 func getPodState(podStateMap elasticsearch.PodStateMap, podStateType elasticsearch.PodStateType) []string {
 	if v, ok := podStateMap[podStateType]; ok {
+		sort.Strings(v)
```

Comment:
since its not explicit in the summary -- is a core issue that the names of the fluentd pods, while remaining the same, if they are populated in a different order will be recognized as being different and therefore cause a status update?

Reply:
just seeing this, but yes, that is what i found